arci-urdf-viz/src/client.rs

```diff
@@ -140,6 +140,20 @@ impl UrdfVizWebClient {
                         .unwrap()
                         .positions;
                     let duration_sec = target.duration.as_secs_f64();
+
+                    if duration_sec == 0.0 {
```

Comment:
I feel bad when compare float using '=='.

I think it is better to use 'is_zero()' to duration instance instead of f64.

```suggestion
                    if target.duration.is_zero() {
```

I know that it's not problem actually.